nova/compute/manager.py

```diff
@@ -681,8 +681,8 @@ def _init_instance(self, context, instance):
                 sys_meta = utils.instance_sys_meta(instance)
                 power_on = sys_meta.get('old_vm_state') != vm_states.STOPPED
 
-                block_dev_info = self._get_instance_volume_block_device_info(
-                            context, instance)
+                block_dev_info = self._get_instance_block_device_info(
+                                         context, instance)
```

Comment:
In fact, my change attempts to fix the pep8 for that particular line. By the way, there are tons of pep8 issues in the original manager.py file and it is existing code.